### PR TITLE
EPMRPP-76316 || Pricing page. Compare plans table for mobile view

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: src/react-components/common/info-with-tooltip/infoWithTooltip.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ InfoWithTooltip.propTypes = {
   className: PropTypes.string,
   children: PropTypes.func,
   tooltip: PropTypes.node.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```
Comment (chivekrodis) [inline src/react-components/common/info-with-tooltip/infoWithTooltip.jsx:65]: As it's not required anymore please add `onClick` to defaultProps

File: src/react-components/plans-block/plansBlock.jsx [hunk 2]
```diff
@@ -88,28 +125,68 @@ const PlansBlock = () => {
...
 
-    const onInfoClick = (title, tooltip) => {
-      if (getIsMobileView()) {
-        showModal(<NotificationModal title={title} description={tooltip} />);
-      }
-    };
+    if (getIsMobileView()) {
+      const onInfoClick = (title, tooltip) => {
+        if (getIsMobileView()) {
+          showModal(<NotificationModal title={title} description={tooltip} />);
```
Comment (chivekrodis) [inline src/react-components/plans-block/plansBlock.jsx:134]: Please check console after opening this notification
Also remove visited styles for `a` tag(open Enterprise features hint)

File: src/react-components/plans-block/plansBlock.jsx [hunk 3]
```diff
@@ -88,28 +125,68 @@ const PlansBlock = () => {
...
+        const rows = titles.map(({ id, name, info }, index ) => {
+          let option = options[id];
+          if (id === 'support') {
+            option = option ? `${option} hours`: '';
+            name = 'Professional Support';
+          }
+
+          return <div key={index} className={cx('plan-row', { disable: !option })}>
+            <i className={cx('row-status')} />
+            {option && option !== true &&
```
Comment (chivekrodis) [inline src/react-components/plans-block/plansBlock.jsx:148]: `option !== true` you use above as `!option`

Comment (Vadim73i) [reply src/react-components/plans-block/plansBlock.jsx:148]: Screenshot 2022-04-19 at 11 24 02 AM

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**